django/contrib/staticfiles/storage.py

```diff
@@ -470,6 +470,8 @@ def load_manifest(self):
             version = stored.get("version")
             if version == "1.0":
                 return stored.get("paths", {})
+            elif version == "1.1":
```

Comment:
Should we return an empty hash for manifests in the 1.0 format?
```python
            if version == "1.0":
                return stored.get("paths", {}), ""
            elif version == "1.1":
                return stored.get("paths", {}), stored.get("hash", "")
```

Reply:
Can we add a test for loading the manifest in version 1.0?

Reply:
ACK to both, the test is a bit hacky since I need to reuse a static file and patch the storage, but okay :)

That said, given the "code" I am wondering if we should just do

```py
return stored.get("paths", {}), stored.get("hash", "")
```
independent of the version. It has a fallback for `paths/hash` either way… not sure what the version check currently offers us there.

Reply:
> independent of the version. It has a fallback for `paths/hash` either way… not sure what the version check currently offers us there.

`ValueError` is raised when the version doesn't match `1.0` or `1.1`. I think it's worth keeping. The error is raised when someone tries to load a "new" manifest with an older version of Django,